requirements.txt

```diff
@@ -6,9 +6,9 @@ basicsr
 blendmodes
 clean-fid
 einops
-fastapi>=0.90.1
+fastapi>=0.111.0
 gfpgan
-gradio==3.41.2
+gradio==4.31.3

```

Comment:
As far as I remember there were problems with Gradio version, have you tested the script with this version?

Reply:
Right, we can hold off this PR from merging for now. I will check for an alternative solution.